deploy/playbook.yml

```
@@ -0,0 +1,73 @@
+- name: Install to-do app in module4 servers
+  hosts: module4
+  vars_prompt:
+    - name: trello_api_key
+    - name: trello_token
+
+  tasks:
+  - name: Install git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: true  
+
+  - name: Install python 3.x
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: true  
+  
+  # create /opt/todoapp
+  # make the folder read/writeable by ec2-user
+  - name: mkdir and chown
```

My instinct says that any comments here are a sign the name isn't clear enough! Comments are so quick to find themselves outdated, I'm a fan of avoiding them wherever you can!
```suggestion
  - name: create /opt/todoapp with appropriate permissions for ec2-user
```

---

Good observation.  The comments are here because I first sketched the file structure with everything I intended to do, then researched each step in ansible docs.  Therefore they are 100% technical debt as the task names should be explicative enough

That's also why this directive has two comments—because I found this directive after starting with actual mkdir and chown.  The name is really me letting myself know that this directive does both.  Your name is waaay better.